core/schema/src/main/resources/org/hawkular/schema/updates/schema-0.19.0.groovy

```diff
@@ -31,4 +31,40 @@ schemaChange {
   author 'jsanda'
   tags '0.19.x'
   cql "ALTER TABLE data WITH compaction = {'class': 'DateTieredCompactionStrategy'} AND default_time_to_live = 604800"
-}
\ No newline at end of file
+}
+
+schemaChange {
+  version '3.2'
+  author 'snegre'
+  tags '0.19.x'
+  cql "ALTER TABLE data DROP aggregates"
```

Comment:
Why are we dropping these? With compression at least we could use these (to get some aggregates of blocks of data)